crates/acir/src/circuit/gate.rs

```diff
@@ -128,6 +129,15 @@ impl std::fmt::Debug for Gate {
                     b.last().unwrap().witness_index(),
                 )
             }
+            Gate::Directive(Directive::ToBytes { a, b, bit_size: _ }) => {
+                write!(
+                    f,
+                    "Split Bytes: {} into x{}...x{}",
```

Comment:
Can you remove reference to Split Bytes here too?

Reply:
Done